Makefile

```diff
@@ -1,21 +1,11 @@
 # simple makefile to simplify repetitive build env management tasks under posix
 
-# caution: testing won't work on windows, see README
-
 PYTHON ?= python
-CYTHON ?= cython
-PYTEST ?= pytest
-
-# skip doctests on 32bit python
-BITS := $(shell python -c 'import struct; print(8 * struct.calcsize("P"))')
-
-all: clean inplace test
 
 clean:
 	$(PYTHON) setup.py clean
 	rm -rf dist
 
-in: inplace # just a shortcut
```

Comment:
would be nice to replace this `in` target with what we do now though. As in, still be able to do `make in`

Reply:
it would be a bit weird that `in` is not the same as `inplace` anymore though ... see https://github.com/scikit-learn/scikit-learn/pull/29296#discussion_r1645676596

Reply:
I would rather rename this to `dev-setuptools` for consistency.

Reply:
Yeah we could do this, although `dev-setuptools` would be different to `dev-meson` in the sense that you need `make dev-setuptools` each time you change a Cython file.

Keeping `make inplace-setuptools` you could argue that if some people are still using `make in`, they can still `make in<TAB>` so slightly less disruptive. But the number of these people is beween 0 and 1 apparently so :shrug:.

I am hoping to look at #29012 again so in the medium term it does not matter that much, so I am fine either way ...

Reply:
I'm also fine either way, we should not bother too much since we expect to have less and less people installing using setuptools.

Reply:
+1 for `dev-setuptools`. Otherwise it's too confusing.

Reply:
dev-setuptools it is, I also reordered the targets in rough order of importance